config/locales/jobseekers.yml

```diff
@@ -74,6 +76,7 @@ en:
             safeguard children.
           heading: Current role and employment history
           no_employments: No employment specified
+          step_title: Employment History
```

Comment:
All of the step titles are exactly the same as the headings in the designs (i.e no extra capitalisation)